test/run

```diff
@@ -216,13 +216,25 @@ test_application() {
 
 test_from_dockerfile(){
   info "Test from Dockerfile"
+  # Due to the old SQLite, the latest LTS release of Django
+  # does not work on RHEL 7. We are testing old Django example
+  # from master branch on EL 7 and the new one from 2.2.x branch
+  # on all other platforms. Also, Django 2 does not support Python 2
+  # so we have to test the older version on newer platforms (EL 8)
+  # with older Python.
```

Comment:
I gotta admit I was a bit confused here, due to the new/old Python, new/old Django, new/old Platform. How about:

> The latest LTS release of Django does not work on RHEL 7 (due to an old version of SQLite), and no longer supports Python 2. So on all CentOS/RHEL 7 images and all Python 2 images, we're using the old Django version 2.2.x. In all other cases we're using the latest LTS release of Django from the main branch of the django-ex project.